docs/source/onedal/build_app/build-application.rst

```diff
@@ -117,13 +117,11 @@ Applications on Windows* OS
 
       * - Static linking
         - 
-          | onedal_core.lib,
           | onedal_core.lib,
           | onedal_thread.lib
       * - Dynamic linking
         - 
           | onedal_core_dll.lib
-          | onedal_core_dll.lib
```

Comment:
@napetrov, should it be replaced with **onedal_dpc_dll.lib**?

Reply:
It's just an example here.  ether can work